From f2ed9951530731ba8ced728a3b26a2a75e6d8bd4 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Tue, 20 Dec 2011 23:30:04 +0000
Subject: cpan/Term-UI: add NAME headings in modules with POD

This fixes the Debian Lintian warning about missing NAME sections in
manpages.

Bug: https://rt.cpan.org/Ticket/Display.html?id=73400
Bug-Debian: http://bugs.debian.org/650452

Patch-Name: fixes/manpage_name_Term-UI.diff
---
 cpan/Term-UI/lib/Term/UI/History.pm |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/Term-UI/lib/Term/UI/History.pm b/cpan/Term-UI/lib/Term/UI/History.pm
index 1d77c01..9da7e47 100644
--- a/cpan/Term-UI/lib/Term/UI/History.pm
+++ b/cpan/Term-UI/lib/Term/UI/History.pm
@@ -8,7 +8,7 @@ use base 'Log::Message::Simple';
 
 =head1 NAME
 
-Term::UI::History
+Term::UI::History - history function
 
 =head1 SYNOPSIS
 
